src/makeCalculator.js

```diff
@@ -4,7 +4,25 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  const calculated = {
+    add: (a, b) => a + b,
+    subtract: (a, b) => a - b,
+    multiply: (a, b) => a * b,
+    divide: (a, b) => a / b,
+    reset: function () {
+      this.result = 0;
+
+      return this;
```

Comment:
[CODE KNOWLEDGE] - When defining methods in an object, you don't need to use the function keyword. Use the method shorthand syntax instead.

BAD EXAMPLE:
reset: function() {
  this.result = 0;
  return this;
},

GOOD EXAMPLE:
reset() {
  this.result = 0;
  return this;
},